src/App.tsx

```diff
@@ -1,61 +1,112 @@
+import { useState } from 'react';
+import { Todo } from './services/types';
+import { PreparedTodo } from './services/types';
+import { TodoList } from './components/TodoList';
+import { userSearcher } from './services/helpers';
+import { generateId } from './services/helpers';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const todos: Todo[] = todosFromServer;
+
+const preparedTodo: PreparedTodo[] = todos.map(todo => ({
+  id: todo.id,
+  title: todo.title,
+  completed: todo.completed,
+  user: userSearcher(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState<string>('');
```

Comment:
if you set default primitive value you don't need to set type to state, it makes under the hood
```suggestion
  const [title, setTitle] = useState('');
```